src/repo/infra.go

```diff
@@ -0,0 +1,12 @@
+package repo
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+
+func Update(cr *sql.DB,table string,condition string,column string,updatedValue string){
```

Comment:
pls look at the code there:
https://github.com/treenq/treenq/blob/main/src/repo/store.go#L56C17-L56C27

you can keep writing in the same find for the same database struct